DotnetViewComponents/ViewComponents/ImageViewComponent.cs

```diff
@@ -0,0 +1,21 @@
+namespace DotnetViewComponents.ViewComponents
+{
+    using Microsoft.AspNetCore.Mvc;
+    using DotnetViewComponents.ViewModels;
+
+    public class ImageViewComponent : ViewComponent
+    {
+        public IViewComponentResult Invoke(
+            string src,
+            string alt,
+            string caption,
+            string? sizes = null,
```

Comment:
This seems like an obscure attribute to include (I don't think I've ever used it) can we cover this in additionalAttributes to make it simpler?